src/providers/cross-chain-sync.provider.ts

```diff
@@ -28,7 +28,7 @@ export interface TempSyncingNetworks {
 }
 
 // all networks that are supported by cross-chain sync feature
-export const supportedNetworks = [
+export const veBalSyncSupportedNetworks = [
```

Comment:
Can you add this as a boolean to config. Maybe under a `veBalSync` section. Can you also add the LayerZeroNetworkId to that section as well. We want to keep all network config in config files so we don't forget about it when adding new networks. 

You can still keep this variable just make it a filter on networks that have the boolean set to true.